mailbox/elasticsearch/src/test/resources/logback-test.xml

```diff
@@ -21,5 +21,7 @@
                 <appender-ref ref="CONSOLE" />
         </logger>
 
+        <logger name="com.datastax.driver.core.QueryLogger.SLOW" level="DEBUG" />
```

Comment:
I put this line in this file because and I do not know if this is normal but when launching jmap cassandra execution test. Logback use this config file. IMO, I think either we should have one and only one logback-test.xml for the all project.
For the moment we have two logback-test.xml and 9 logback.xml:
server/data/data-ldap-integration-testing/src/test/resources/logback.xml
server/mailet/mailets/src/test/resources/logback-test.xml
server/mailet/integration-testing/src/test/resources/logback.xml
server/protocols/jmap-integration-testing/memory-jmap-integration-testing/src/test/resources/logback-test.xml
server/container/guice/memory-guice/sample-configuration/logback.xml
server/container/guice/cassandra-guice/src/main/resources/logback.xml
server/container/guice/jpa-guice/src/main/resources/logback.xml
dockerfiles/run/guice/jpa/destination/conf/logback.xml
dockerfiles/run/guice/cassandra-ldap/destination/conf/logback.xml
mpt/impl/imap-mailbox/core/src/main/resources/logback.xml
mailbox/elasticsearch/src/test/resources/logback-test.xml
I have read that logback-test.xml have priority one logback.xml.
But when there is multiple file with same name/url from multiple jar, the first jar of the classpath have priority over the other.
My question is how do we know the order of the different maven project in the classpath ? Do we really control that properly ?